src/Writers/html.jl

```diff
@@ -0,0 +1,27 @@
+"""
+Provides the `render()` methods to write the documentation as HTML files
+(`MIME"text/html`).
```

Comment:
Thanks for catching the typo. However, I would prefer not to escape, since it's a GitHub bug, and not an issue in, say, Atom. And as long as we have the closing `"` present as well, the highlighting doesn't break for the whole file, so I think we could live with it.

I also reported the highlighting bug to GitHub just now.